### DISCUSSION TITLE
Empty News lists

### DISCUSSION THREAD

**Discussion by ellocoquecorre** [e107inc/e107#5296] [General]
When I create a new item, I get a message saying that it was successfully added (Success - Successfully created #9) but in both List/News, and List/Submitted News, I see nothing.

I can see the preview of the post, it shows me the path to the post, for example "News / Category 1 / Post #9", but when I go to "News", it tells me there is nothing to display.

**Comment by Jimmi08** [comment]
- what is your e107 version and what theme are you using? PHP version? 

Check location for single news:
![image](https://github.com/user-attachments/assets/7192780c-9c37-47a9-b809-bc45acad5666)

Check default template in news prefs:
![image](https://github.com/user-attachments/assets/9166f078-a995-4149-9835-7796e7443793)

**Comment by ellocoquecorre** [comment]
PHP Version 8.2.12
e107 v2.3.3
XAMPP Version 8.2.12

I've tried every single theme, and still nothing.

**Reply by tgtje** [reply]
same local version : can not replicate. 
Do mind ; submitted news is for external ( as a post by user) message, so in admin created it would not show in Submitted news..

**Comment by Vodhin** [comment]
Check `Start & End` are blank (no 0 value), and `Visibility` is` Everyone `(or whomever you want) ?
